core/base/src/utils/misc.ts

```diff
@@ -6,3 +6,22 @@ export function lazyInstantiate<T>(factory: () => T): () => T {
     return instance;
   }
 }
+
+export function onlyOnce<T extends []>(fn: (...args: T) => any, ...args: T): () => void {
```

Comment:
intedesting

Reply:
This is for all the factory registration try - catch stuff. I still have it only locally. I'd really like to figure out why the same module code is ever executed twice. This sounds like the real issue to me.